dev-infra/pullapprove/group.ts

```diff
@@ -34,7 +34,7 @@ const CONDITION_TYPES = {
   INCLUDE_GLOBS: /^contains_any_globs/,
   EXCLUDE_GLOBS: /^not contains_any_globs/,
   ATTR_LENGTH: /^len\(.*\)/,
-  GLOBAL_APPROVAL: /^global-(docs-)?approvers not in groups.approved$/,
+  GLOBAL_APPROVAL: /^\"global-(docs-)?approvers\" not in groups.approved$/,
```

Comment:
Nit: You don't need to escape `"`:

```suggestion
  GLOBAL_APPROVAL: /^"global-(docs-)?approvers" not in groups.approved$/,
```